ArduCopter/Log.cpp

```diff
@@ -484,18 +502,82 @@ const struct LogStructure Copter::log_structure[] = {
       "DU32",  "QBI",         "TimeUS,Id,Value", "s--", "F--" },
     { LOG_DATA_FLOAT_MSG, sizeof(log_Data_Float),         
       "DFLT",  "QBf",         "TimeUS,Id,Value", "s--", "F--" },
+    
+// @LoggerMessage: HELI
+// @Description: Helicopter related messages 
+// @Field: TimeUS: Microseconds since system startup
+// @Field: DRRPM: Desired rotor speed
+// @Field: ERRPM: Estimated rotor speed when no speed sensor or governor is used
+// @Field: Gov: Governor Output
+// @Field: Throt: Logic controlled output
 #if FRAME_CONFIG == HELI_FRAME
     { LOG_HELI_MSG, sizeof(log_Heli),
       "HELI",  "Qffff",        "TimeUS,DRRPM,ERRPM,Gov,Throt", "s----", "F----" },
 #endif
+
+// @LoggerMessage: PL
+// @Description: Prescion Landing messages
+// @Field: TimeUS: Microseconds since system startup
+// @Field: Heal: True if Prescion Landing is healthy
+// @Field: TAcq: True if Landing Target is detected
+// @Field: pX: Target position relative to vehicle, X-Axis (0 if target not found)
+// @Field: pY: Target position relative to vehicle, Y-Axis (0 if target not found)
+// @Field: vX: Target velocity relative to vehicle, X-Axis (0 if target not found)
+// @Field: vY: Target velocity relative to vehicle, Y-Axis (0 if target not found)
+// @Field: mX: Target's relative postion to vehicle as 3-D Vector, X-Axis
+// @Field: mY: Target's relative postion to vehicle as 3-D Vector, Y-Axis
+// @Field: mZ: Target's relative postion to vehicle as 3-D Vector, Z-Axis
+// @Field: LastMeasUS: Time when target was last detected
+// @Field: EKFOutl: EKF's outliner count (3 consecutive outliers lead to EKF accepting updates)
+// @Field: Est: Type of estimator used
+
 #if PRECISION_LANDING == ENABLED
     { LOG_PRECLAND_MSG, sizeof(log_Precland),
       "PL",    "QBBfffffffIIB",    "TimeUS,Heal,TAcq,pX,pY,vX,vY,mX,mY,mZ,LastMeasUS,EKFOutl,Est", "s--ddmmddms--","F--00BB00BC--" },
 #endif
+
+// @LoggerMessage: SIDD
+// @Description: System ID information
+// @Field: TimeUS: Microseconds since system startup
+// @Field: Time: Time reference for waveform
```

Comment:
Gah!  We calculate this by accumulating `G_Dt`.  Accumulated floating point errors may be an issue.  This... probably isn't your problem ;-)